relentless/simulate/lammps.py

```diff
@@ -0,0 +1,120 @@
+import abc
+import os
+from packaging import version
+
+import numpy as np
+
+from relentless.core.volume import TriclinicBox
+from . import simulate
+
+try:
+    import lammps
+    _lammps_found = True
+except ImportError:
+    _lammps_found = False
+
+class LAMMPS(simulate.Simulation):
+    def __init__(self, operations, **options):
+        if not _lammps_found:
+            raise ImportError('LAMMPS not found.')
+
+        super().__init__(operations,**options)
+
+    def _new_instance(self, ensemble, potentials, directory):
+        sim = super()._new_instance(ensemble,potentials,directory)
+
+        # create lammps instance with all output disabled
+        #quiet_launch = ['-echo','none',
+        #                '-log','none',
+        #                '-screen','none',
+        #                '-nocite']
+        sim.lammps = lammps.lammps()#cmdargs=quiet_launch)
```

Comment:
We will want to uncomment these for production, but your testing issues point out that it may not always be good to run LAMMPS in "quiet" mode. Maybe we should consider adding an argument to the constructor that when true (default) uses these arguments, and maybe when false allows only output to console? The argument could be called `quiet` if it works like this or `verbose` if you negate the behavior?